folium/features.py

```diff
@@ -652,6 +682,69 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
         self.className = class_name
 
 
+class Tooltip:
+    """
+    Creates a Tooltip object for adding to features to display text as a property a Map by executing a javascript
```

Comment:
folium uses a line width of 80 characters, please adhere to it in your code. Are you using Pycharm? I recommend to set the line width in your IDE to warn you when you go over it. Your IDE should also warn for lint errors.

Reply:
I was, but had switched over to VS Code and committed through the web portal to help out with the JS syntax highlighting and brace closures in the template, as keeping it organized with the Jinja2 curly braces expressions, JS template literal embeds was a little challenging (for me). Running a refactor in PyCharm through the VCS now - thanks for your patience and suggestions!

Reply:
Thanks! Looking through the repo and other classes, I see several other code >80 character width violations. I've resolved the issues within my python code, but getting the Jinja2 template to comply with all the conditional assertions and embeds within the JS is challenging. Would the template be an acceptable violation, in order to keep it thematically organized by line and indentation so that it is easier to read/modify for future contributors?